nsv13/code/modules/overmap/weapons/plasma_gun.dm

```diff
@@ -0,0 +1,466 @@
+/obj/machinery/ship_weapon/plasma_caster
+	name = "\improper Magnetic Phoron Acceleration Caster"
+	icon = 'nsv13/icons/obj/plasma_gun.dmi' //This sprite cost me $150
+	icon_state = "plasma_gun"
+	desc = "An MPAC, a strong deterrent used by the Dominion as a method of staving off attackers. It requires a lot of maintenance."
+	anchored = TRUE
+	max_integrity = 1200 //It has no chance to survive a self-induced plasmafire
```

Comment:
```suggestion
	max_integrity = 1200 //It has no chance to survive a self-induced plasmafire
	obj_integrity = 1200
```
Probably want to make sure it actually has that much hit points as well, instead of just increasing the capacity.